.travis.yml

```diff
@@ -57,3 +57,6 @@ jobs:
       # Built steps inherited from the default stage that we don't want
       before_install: ignore
       after_success: []
+  allow_failures:
+    - python: '3.6'
+      env: TEST=build
```

Comment:
I don't think this environment variable is required (unless I'm missing something), it was just helpful on molo core to split the CI so that each bit could pass individually.

Reply:
Ah right! I think I just added this temporarily while I fixed up all the python things. I'm actually going to remove `allow_failures` entirely, as everything should now be passing with Python3